lib/moonshot/ssh_fork_executor.rb

```diff
@@ -9,7 +9,8 @@ def run(cmd)
       output = StringIO.new
 
       exit_status = nil
-      Open3.popen3(cmd) do |_, stdout, _, wt|
+      Open3.popen3(cmd) do |_, stdout, stderr, wt|
```

Comment:
Adding this makes this PR modify moonshot internals and not just add a new plugin. Not sure if we want that.
Main concern being it makes makes merging into master when we can little more complicated compared to only adding a plugin.

Reply:
I'm not sure we just want to add stderr to the "output" property of the result object.  It would probably be better to modify the Result struct to add stderr as a separate property.

Reply:
Thank you for the suggestion @pdrakeweb